fi/index.html

```diff
@@ -0,0 +1,196 @@
+    <link rel="stylesheet" href="/static/external/css/reset.css" type="text/css" media="screen" />
+    <link rel="stylesheet" href="/static/external/css/responsive-gs-12col.css" type="text/css" media="screen" />
+    <link rel="stylesheet" href="/static/css/base.css" type="text/css" media="screen" />
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:300,400,700|Gentium+Basic" rel="stylesheet" type="text/css" />
+    <link rel="icon" href="/static/SymPy-Favicon.ico" />
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
+    <script type="text/javascript" src="/static/javascript/main.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1><a href="index.html"> <img src="/static/images/logo.png" alt="SymPy Logo" /> SymPy</a></h1>
+        <div id="mobile-menu"><i class="icon-ellipsis-vertical"></i></div>
+      </header>
+      <nav id="main-navigation" class="row">
+        
+        
+        <ul>
+          <li class="active"><a href="index.html">Pääsivu</a></li>
+          <li class=""><a href="features.html">Ominaisuudet</a></li>
+          <li class=""><a href="download.html">Lataa</a></li>
+          <li class=""><a href="http://docs.sympy.org">Dokumentaatio <i class="icon-external-link-sign"></i></a></li>
+          <li class=""><a href="support.html">Tuki</a></li>
+          <li class=""><a href="development.html">Kehitys</a></li>
+          <li class=""><a href="donate.html">Lahjoita</a></li>
+          <li><a href="http://live.sympy.org/">Online-komentokehoite <i class="icon-external-link-sign"></i></a></li>
+        </ul>
+        
+      </nav>
+      <section class="row" id="main">
+        <div class="col span_7 clr small_span_4">
+          
+
+<section id="banner">
+  <h2 class="title">Tietoja</h2>
+  <div class="inner">
+    <p class="first">
+      SymPy on symboliseen matematiikkaan tarkoitettu Python-kirjasto. Se pyrkii
+      tulemaan täysien omainsuuksien laskentaohjelmistoksi (CAS) pitäen koodin
+      mahdollisimman yksinkertaisena ollakseen ymmärrettävä ja helposti laajennettava. SymPy
+      on kirjoitettu kokonaan Python-ohjelmointikielellä eikä vaadi ylimääräisiä kirjastoja.
+    </p>
+
+    <section id="get-started">
+    <p><a href="http://docs.sympy.org/latest/tutorial/index.html">Pääse alkuun tutoriaaleilla</a> <a href="https://github.com/sympy/sympy/releases">Lataa nyt</a></p>
+  </div>
+</section>
+
+
+          <article id="content">
+            
+
+<h2 class="title">Miksi SymPy</h2>
+<div class="inner">
+    <p>SymPy on&hellip;</p>
+    <ul>
+        <li><strong>Vapaa:</strong> Lisensioitu BSD-lisenssillä, SymPy on vapaa ja ilmainen</li>
+        <li><strong>Python-ohjelmointikieleen perustuva:</strong> SymPy on kirjoitettu kokonaan Pythonilla ja käyttää Pythonia kielenään</li>
+        <li><strong>Kevyt:</strong> Ainoa ulkoinen kirjasto jonka SymPy tarvitsee on NumPy, joten se on helppo käyttää.</li>
```

Comment:
Actually, the external library was mpmath. NumPy is useful but not necessary for using SymPy.